src/test/e2e/42_values_test.go

```diff
@@ -53,7 +53,15 @@ func TestValues(t *testing.T) {
 	require.NoError(t, err, "unable to get raw template configmap")
 	require.Contains(t, kubectlOut, "template={{ .shouldNotBeProcessed }}")
 
-	// Remove the package
-	stdOut, stdErr, err = e2e.Zarf(t, "package", "remove", "test-values", "--confirm")
+	// Remove the package with values
+	valuesFile := filepath.Join(src, "values", "values.yaml")
+	stdOut, stdErr, err = e2e.Zarf(t, "package", "remove", "test-values", "--confirm", "--features=\"values=true\"", "--values", valuesFile, "--set-values", "removeKey=custom-remove-value")
 	require.NoError(t, err, stdOut, stdErr)
+
+	// Verify the remove actions used the values correctly
+	// Check that the default value from values.yaml was templated
+	require.Contains(t, stdOut, "REMOVE_TEST_VALUE=default-value", "remove action should have templated default value from values.yaml")
```

Comment:
Should we pass the `override-values.yaml` file into remove? This doesn't check that the default values are templated since we're passing them in.